src/diffusers/models/unets/unet_2d_blocks.py

```diff
@@ -2440,7 +2477,6 @@ def forward(
         attention_mask: Optional[torch.FloatTensor] = None,
         encoder_attention_mask: Optional[torch.FloatTensor] = None,
     ) -> torch.FloatTensor:
-        lora_scale = cross_attention_kwargs.get("scale", 1.0) if cross_attention_kwargs is not None else 1.0
```

Comment:
if cross_attention_kwargs is not None:
            if cross_attention_kwargs.get("scale", None) is not None:
                logger.warn("Passing `scale` to `cross_attention_kwargs` is depcrecated. `scale` will be ignored.")